components/studio/bin/hab-studio.sh

```diff
@@ -448,6 +452,17 @@ new_studio() {
         $bb mount $v --bind "$ARTIFACT_PATH" "$studio_artifact_path"
       fi
     fi
+
+    # Mount ${CERT_PATH} under '/hab/cache/ssl' in the Studio,
+    # unless `$NO_CERT_PATH` is set
+    if [ -z "${NO_CERT_PATH}" ]; then
+      studio_cert_path="${HAB_STUDIO_ROOT}${HAB_CACHE_CERT_PATH}"
+      if ! $bb mount | $bb grep -q "on $studio_cert_path type"; then
```

Comment:
I think this will be susceptible to the busybox/musl bug described in https://github.com/habitat-sh/habitat/issues/6591#issuecomment-496564865 and https://github.com/habitat-sh/habitat/pull/6608

We should instead use   `$bb cut -d' ' -f2 /proc/mounts | $bb grep -q -x "$studio_cert_path" ` (and verify I wrote this correctly).

Reply:
This is code copied from the artifacts mount - I don't want to touch the artifacts flow in this PR.  We should do a fix for all the bb mounts separately I think.